kiwi/schema/kiwi.rnc

```diff
@@ -1588,7 +1588,7 @@ div {
         attribute efifatimagesize { xsd:nonNegativeInteger }
         >> sch:pattern [ id = "efifatimagesize" is-a = "image_type"
             sch:param [ name = "attr" value = "efifatimagesize" ]
-            sch:param [ name = "types" value = "oem" ]
+            sch:param [ name = "types" value = "iso" ]
```

Comment:
I think the way you set the schematron rule will overwrite it, so it should be set as

```
sch:param [ name = "types" value = "oem iso" ]
```

Reply:
Thanks!
I'll add the suggestion to the PR

Reply:
Thanks, any news on this one ?

Reply:
Took me a little time to properly test the change. I've uploaded another commit with your suggestion